okapi/settings.php

```diff
@@ -180,6 +185,21 @@
          * https://help.github.com/articles/creating-an-access-token-for-command-line-use/
          */
         'GITHUB_ACCESS_TOKEN' => null,
+
+        /**
+         * Maximum 'resolution' of saved images in pixels
+         */
+        'IMAGE_MAX_PIXEL_COUNT' => 524288,  # 0.5 MP
+
+        /**
+         * Quality of saved JPEG images on a scale from 50 to 100. The higher the
+         * quality, the sharper the image and larger the saved image files
+         * (file size grows exponentially). The PHP default is 75, which is also
+         * used in OCPL code. As OKAPI always resamples uploaded images - which
```

Comment:
OKAPI always reasamples - why? :) Probably there's a thread for this, if so, I will find it in a minute.